### ISSUE TITLE
Convert from 2018 to 2017

### ISSUE THREAD

**Issue by guohengkai** [leonid-pishchulin/poseval#16]
Hi,

I'm trying to evaluate the results with 2017 format on 2018 eval json. The convert function does not merge the annotations in the same image, so the frame number of predictions and ground truths do not match.

**Comment by leonid-pishchulin** [comment]
Prediction must be provided in the same format as the ground truth. Predictions have to be provided per sequence, prediction files must be named the same as the ground truth files.

**Comment by guohengkai** [comment]
@leonid-pishchulin Yes, the names are the same. And in your codes, the 2018 json will be converted into 2017 format. But after conversion, the data is wrong because it do not merge multiple poses from one image into single annotations.

**Comment by leonid-pishchulin** [comment]
could you send me your predictions so that I can see what's happening?

**Comment by guohengkai** [comment]
@leonid-pishchulin Sent.
